base/i18n/break_iterator_unittest.cc

```diff
@@ -140,9 +140,11 @@ TEST(BreakIteratorTest, BreakWordThai) {
 // dictionary to detect word boundaries in Thai, Chinese, Japanese, Burmese,
 // and Khmer. Due to the size of such a table, the part for Chinese and
 // Japanese is not shipped on mobile.
+// Cobalt does not support Chinese/Japanese word breaking yet. This feature
+// requires a big dictionary(cjdict.txt) to support.
 #if !(BUILDFLAG(IS_IOS) || BUILDFLAG(IS_ANDROID))
```

Comment:
Wouldn't it be cleaner to add `|| BUILDFLAG(IS_STARBOARD)` in places like these ? It would create a smaller upstream diff.

Reply:
It would be cleaner, but I think more difficult to track disabled tests unless you have a good solution for doing that with macros. I figured doing it with the gtest DISABLED macro would allow us to easily see what is and isn't disabled (alongside a single list of sources for each suite that isn't compiled)